commit f40c562855294bf4e7268274d7461dc32c1e6b25
Author: Steve French <sfrench@us.ibm.com>
Date:   Wed Jun 28 00:13:38 2006 +0000

    [CIFS] Fix authentication choice so we do not force NTLMv2 unless the
    user specifies it is required or turns of ntlm
    
    Signed-off-by: Steve French <sfrench@us.ibm.com>

diff --git a/fs/cifs/sess.c b/fs/cifs/sess.c
index b7d49c03985c..7202d534ef0b 100644
--- a/fs/cifs/sess.c
+++ b/fs/cifs/sess.c
@@ -323,11 +323,12 @@ CIFS_SessSetup(unsigned int xid, struct cifsSesInfo *ses, int first_time,
 	__u16 action;
 	int bytes_remaining;
 
-	cFYI(1,("new sess setup"));
 	if(ses == NULL)
 		return -EINVAL;
 
 	type = ses->server->secType;
+
+	cFYI(1,("sess setup type %d",type));
 	if(type == LANMAN) {
 #ifndef CONFIG_CIFS_WEAK_PW_HASH
 		/* LANMAN and plaintext are less secure and off by default.